### DISCUSSION TITLE
migrazione a doker, problema con caddy

### DISCUSSION THREAD

**Discussion by giacomo-git** [indomus/forum#1153] [HUB: Home Assistant]
Ciao
sto seguendo la guida per migrare da venv a docker
sto provando a configurare caddy
eseguendo :
ls -latr /home/pi/caddy/data/caddy/certificates/acme-staging-v02.api.letsencrypt.org-directory/xxx.duckdns.org
ottengo
ls: cannot access '/home/pi/caddy/data/caddy/certificates/acme-staging-v02.api.letsencrypt.org-directory/xxx.duckdns.org': Permission denied

eseguendo il comado come sudo vedo i cerificati, ma come owner vedo root.

-rw------- 1 root root  227 Dec  7 12:42 xxx.duckdns.org.key
-rw------- 1 root root  165 Dec  7 12:42 xxx.duckdns.org.json
-rw------- 1 root root 4191 Dec  7 12:42 xxx.duckdns.org.crt

Nella guida invece vedo che come owner appare pi

-rw------- 1 pi pi  227 gen  1 00:00 casamia.duckdns.org.key
-rw------- 1 pi pi  171 gen  1 00:00 casamia.duckdns.org.json
-rw------- 1 pi pi 3299 gen  1 00:00 casamia.duckdns.org.crt

Sapreste dirmi dove mi sono perso?
Grazie

**Comment by snakuzzo** [comment]
Va bene così com'è...non preoccuparti
I certificati vengono generati come root.

**Reply by giacomo-git** [reply]
Ok grazie.
nel frattempoperò ho aggiunto pi come owner.
Cmq sono riuscito a configuare il reverse proxy e la migrazione sembra essere terminata con successo

Spero non sia un problema aver aggiunto pi come owner
grazie

**Reply by snakuzzo** [reply]
inutile cambiare l'owner...i certificati quando vengono rinnovati vengono generati da root (root del container per la precisione) che vedrai come root su FS dell'host.
Comunque no...non è un problema aver cambiato l'owner

**Comment by giacomo-git** [comment]
perfetto

non so se questo è il tred corretto cmq ci provo :) 
esiste un modo per migrare deCONZ su docker?
oppure è sufficente seguire la guida https://indomus.it/guide/come-installare-e-configurare-deconz-con-docker-su-raspbian-di-raspberry-pi/
e poi disistallare deconz su venv?

**Reply by snakuzzo** [reply]
puoi seguire quella guida
